src/cache/raw.rs

```diff
@@ -56,6 +56,17 @@ pub(crate) struct Class {
     pub(crate) members_by_params_offset: u32,
     /// The number of member-by-params entries for this class.
     pub(crate) members_by_params_len: u32,
+    /// Whether this class was synthesized by the compiler.
+    ///
+    /// `0` means `false`, all other values mean `true`.
+    pub(crate) is_synthesized: u32,
```

Comment:
maybe turn this into `is_synthesized: u8`, and add `_reserved: [u8; 3]` to the end.